roles/lead_engineer.md

```diff
@@ -0,0 +1,91 @@
+
+
+### Make sensible and well justified technical architecture decisions, involving the team in the decision making process where appropriate
+
+A Lead Engineer is expected to be able to design and implement appropriate technical architectures to solve problems. A Lead Engineer is expected to appropriately involve the team in the decision making process to help them develop their technical architecture skills.
+
+
+### Lead workshop and roadmapping sessions to understand customer requirements and convert these in to deliverable iterations
+
+A Lead Engineer is expected to lead workshop and roadmapping sessions and foster collaboration with the wider team to identify a technology roadmap to solve business problems.
+
+
+### Quickly become a trusted partner to the customer team, building a friendly relationship
+
+A Lead Engineer is expected to quickly become a trusted partner to the customer team - both our day-to-day contacts, and more senior stakeholders. A Lead Engineer should be able to put a customer at ease by engaging in small talk, and by building strong and friendly working relationships. A Lead Engineer is expected to be able to credibly represent Made Tech when conversing with senior stakeholders.
+
+
+### Proactively identify opportunities where we can widen the impact of our work with the customer organisation
+
+A Lead Engineer is expected to proactively seek opportunities where we can extend our remit with the customer organisation. A Lead Engineer is expected to understand the objectives of our stakeholders, and to sell Made Tech’s wider capabilities.
+
+
+### Able to make sensible, well reasoned commercial decisions
+
+A Lead Engineer is expected to proactively make good commercial decisions. A Lead Engineer is expected to identify and course correct potential delivery issues before they become impactful. A Lead Engineer is expected to proactively escalate larger delivery issues to the Delivery Manager or Delivery Director.
+
+
+### To have attained the Made Tech Core Engineering Competencies
+
+A Lead Engineer is expected to be an expert in their craft, practising and coaching others in the Made Tech Core Engineering Competencies.
+
+
+## On Engagements without a Delivery Manager:
+
+### Be the primary point of contact for the customer
+
+A Lead Engineer is expected to naturally position themselves as the primary day to day point of contact for the customer. A Lead Engineer is expected to gain the trust of the customer such that they’re seen as the first port of call for all customer contacts.
+
+
+### Ensure customer expectations are exceeded given the budget available
+
+A Lead Engineer is expected to understand the financial constraints, as well as the customer objectives for the delivery. A Lead Engineer is accountable for ensuring the customer’s expectations are exceeded for the available budget.
+
+
+### To be ultimately accountable for the successful delivery of customer engagements
```

Comment:
Are we saying this is the Delivery Manager is accountable for this? Not sure this is the right split

Reply:
This is on engagements *without* a DM. What do you think the right split is?

Reply:
Well that's my point, I'd have said that `delivery` itself can only be actioned by the people doing the work i.e. cutting the code.

 A delivery manager - someone who is accountable/responsible for lining up the surroundings of the work cannot be ultimately responsible/accountable for the cutting of code.

Maybe Lead Engineer is always ultimately accountable for successful delivery

Reply:
Is it reasonable to see that cutting code is an important part of the delivery, but far from the only thing that matters?

Maybe it needs clarification that Leads are always ultimately responsible for the successful technical implementation, and on deliveries without a DM are further responsible for the overall success of the delivery?

Reply:
I'd have the same expectations no matter what team. Otherwise Lead means different things in different contexts.

Why can't we have Lead Engineer being ultimately responsible even with DM?

Reply:
I'm not actually averse to that - though we'll need to redefine the DM role to be clear on this split (the current DM role is clear on accountability for this)